requirements/requirements_asr.txt

```diff
@@ -5,13 +5,15 @@ ipywidgets
 jiwer
 kaldi-python-io
 kaldiio
+lhotse>=1.19.2
 librosa>=0.10.0
 marshmallow
 matplotlib
 packaging
 pyannote.core
 pyannote.metrics
 pydub
+pyloudnorm
```

Comment:
Where this library has been used?

Reply:
It's an optional dependency of Lhotse. We use it to bring the concatenated utterances to the same sound level in dB if you enable CodeSwitchingDataset-like functionality with Lhotse dataloader, specifically here under `normalize_loudness`:

https://github.com/NVIDIA/NeMo/blob/b3cfc8ad15a185572962c9fb74552b6403bd6e11/nemo/collections/common/data/lhotse/dataloader.py#L195-L209

Reply:
Its a pip installable library right? Should be ok. If its c-compiled or has issues on mac/windows we may need to consider dropping it from hard requirements (guarding it)

Reply:
It's pip installable and 100% Python.